ts/input/tex/ams/AmsMethods.ts

```diff
@@ -118,11 +119,79 @@ AmsMethods.Multline = function (parser: TexParser, begin: StackItem, numbered: b
   item.arraydef = {
     displaystyle: true,
     rowspacing: '.5em',
-    columnwidth: '100%',
+    columnspacing: '100%',
     width: parser.options['multlineWidth'],
     side: parser.options['tagSide'],
-    minlabelspacing: parser.options['tagIndent']
+    minlabelspacing: parser.options['tagIndent'],
+    framespacing: parser.options['multlineIndent'] + ' 0',
+    frame: '',   // Use frame spacing with no actual frame
+    'data-width-includes-label': true // take label space out of 100% width
```

Comment:
Why give this a data attribute name, although it is never really used as data attribute?

Reply:
These values are added to the generated `<mtable>` element as attributes, and since this is not a defined attribute from the spec, I am using `data-` for it.  So it actually *is* used as a data attribute.  For example,

``` latex
\begin{multline}
a+b+c\\
d+e+f+g+h\\
i+j
\end{multline}
```

produces

``` xml
<math xmlns="http://www.w3.org/1998/Math/MathML" indentalign="left" display="block">
  <mtable displaystyle="true" rowspacing=".5em" columnspacing="100%" width="100%" framespacing="1em 0" frame="" data-width-includes-label="true">
    <mtr>
      <mtd columnalign="left">
        <mi>a</mi>
        <mo>+</mo>
        <mi>b</mi>
        <mo>+</mo>
        <mi>c</mi>
      </mtd>
    </mtr>
    <mtr>
      <mtd>
        <mi>d</mi>
        <mo>+</mo>
        <mi>e</mi>
        <mo>+</mo>
        <mi>f</mi>
        <mo>+</mo>
        <mi>g</mi>
        <mo>+</mo>
        <mi>h</mi>
      </mtd>
    </mtr>
    <mtr>
      <mtd columnalign="right">
        <mi>i</mi>
        <mo>+</mo>
        <mi>j</mi>
      </mtd>
    </mtr>
  </mtable>
</math>
```

(scroll to see the `data-width-includes-label` attribute on the `mtable`)

Also, note in `ts/output/common/Wrappers/mtable.ts` below where the `data-width-include-label` attribute is read from the `mtable` element at lines 445 and 523.  Making it an attribute means that it will be preserved when cycled through SRE.